config-model/src/main/java/com/yahoo/vespa/model/container/ContainerCluster.java

```diff
@@ -304,6 +307,38 @@ public void addJaxProviders() {
         addSimpleComponent(XPathFactoryProvider.class);
     }
 
+    public void addIdentity() {
+        if (getDeploymentSpec().isPresent()) {
+            DeploymentSpec deploymentSpec = getDeploymentSpec().get();
```

Comment:
These two lines can be replaced by 
```
getDeploymentSpec().map(deploymentSpec -> {
   ...
})
```

Reply:
I know, just didn't feel right at once, but OK